src/Development/Shake/Command.hs

```diff
@@ -219,19 +220,31 @@ parseFSAT = mapMaybe (f . wordsBy (== ':')) . lines
           f _ = Nothing
 
 
-unixWhitelist :: IO [FilePath]
-unixWhitelist = do
+unixWhitelist :: [FilePath]
+unixWhitelist = unsafePerformIO $ do
     home <- getEnv "HOME"
```

Comment:
Should this be [`getHomeDirectory`](https://hackage.haskell.org/package/directory-1.2.4.0/docs/System-Directory.html#v:getHomeDirectory)?